bspump/http/web/server.py

```diff
@@ -268,9 +269,10 @@ def __init__(self, *args, **kwargs):
         self.default = self.default or False
 
     def inner_html(self, default="", readonly=False):
+        readonly_attr = "disabled" if readonly else ""
         return f"""
-            <input type="checkbox" {"checked" if default == True or (default and default.lower() in ("true", "t")) else ""} class="{self.default_classes}" {self.default_input_props}>
-        """
+                    <input type="checkbox" {"checked" if default == True or (default and default.lower() in ("true", "t")) else ""} class="{self.default_classes}" {self.default_input_props} {readonly_attr}>
+                """
```

Comment:
While this is a fine change, I prefer if each commit only contains one change. I want to merge this now so it is ready for Tomáš but in the future it is better to split unrelated changes into different commits.